docs/guide/installation/prod.vuestorefront.io

```diff
@@ -38,11 +38,9 @@ server {
 	gzip_types
 		text/css
 		text/javascript
-		text/xml
```

Comment:
Why was this type also removed? It's not compressed by default, says nginx doc: https://nginx.org/en/docs/http/ngx_http_gzip_module.html#gzip_types

Reply:
Ah, my mistake. Only text/html is compressed by default.

Reply:
so should or shouldn't this be resolved?